README.md

```diff
@@ -465,6 +469,31 @@ If the daemon has been installed, live stats of CPU/system load monitoring and o
 
 `auto-cpufreq --stats`
 
+## Battery charging thresholds
+
+As of v2.2.0, battery charging thresholds can be set in the config file. This enforces your battery to start and stop charging at defined values
+
+### Supported devices
+
+* **Lenovo ThinkPad** (thinkpad_acpi)*
+* **Lenovo IdeaPad** (ideapad_acpi)*
+
+***Please note, your laptop must have an installed ACPI kernel driver specific to the manufacturer.** To check if you have the correct module installed and loaded run `lsmod [module]`
+
+**To request that your device be supported, please open an [issue](https://github.com/AdnanHodzic/auto-cpufreq/issues/new). In your issue, make us aware of the driver that works with your laptop**
```

Comment:
Let's turn this into a link as well: [of the driver](https://github.com/AdnanHodzic/auto-cpufreq/#troubleshooting)

Reply:
I think there's some confusion here. The driver in this case, refers to the acpi driver that allows the kernel to interact with battery, bluetooth, wifi radio, etc... so like thinkpad_acpi and ideapad_acpi, not the CPU scaling driver

Reply:
Oh, right, then let's disregard this one.